aspnetcore/grpc/httpapi.md

```diff
@@ -91,11 +91,65 @@ The client receives three line-delimited JSON objects:
 
 Note that the `WriteIndented` JSON setting doesn't apply to server streaming methods. Pretty printing adds new lines and whitespace to JSON, which can't be used with line-delimited JSON.
 
-### OpenAPI support
+## Experimental OpenAPI support
 
-Transcoding currently doesn't support OpenAPI. During the development of .NET 7, the .NET team will investigate the best way to support OpenAPI.
+There is *experimental* support for generating OpenAPI from gRPC transcoded RESTful APIs. The NuGet package integrates with [Swashbuckle](xref:tutorials/get-started-with-swashbuckle) and is experimental in .NET 7 to give us time to explore the best way to provide OpenAPI support.
```

Comment:
It might help to name the NuGet package alluded to here.  At first it wasn't clear to me if we were talking about [Microsoft.AspNetCore.Grpc.JsonTranscoding](https://www.nuget.org/packages/Microsoft.AspNetCore.Grpc.JsonTranscoding) or [Microsoft.AspNetCore.Grpc.Swagger] as the package that integrates.  But, maybe everyone else would clue into that right away and it is just me.